community/contributing/docs/docs-style-guide.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: single
+title:  "Documentation Style Guide"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+
+This page gives writing style guidelines for the Spinnaker documentation. Since these are guidelines, not rules, use your best judgment when creating content. Feel free to propose changes to this document in a pull request.
+
+If you have a style question that isn't answered in this guide, ask the SIG Docs team in the `#sig-documentation` channel.
```

Comment:
This assumes the reader/contributor is already familiar with the Spinnaker Slack workspace. Is that a safe assumption? (It may well be; I'm just wondering.)